.kitchen.yml

```diff
@@ -82,3 +82,22 @@ suites:
     encrypted_data_bag_secret_key_path: "test/integration/forwarder/encrypted_data_bag_secret"
     run_list:
       - recipe[elkstack::forwarder]
+
+<% if ENV['RACKSPACE_MONITORING_USERNAME'] && ENV['RACKSPACE_MONITORING_API_KEY'] %>
```

Comment:
I'm wondering if failed builds might actually trigger tickets in the queue, if they get far enough in the build to setup the monitoring agent. I'd hate to have tickets show up for this. It's too bad we don't have a way to do a configtest or something for the agent.